### PR TITLE
Add two more libcxx tests that build with clang

### PR DESCRIPTION
**Pull request by anitagov**
Enabled 2 libcxx tests to build and PASS with clang. These were unsupported earlier as they do not build with GCC due to invoking unsupported functions.
3rdparty/libcxx/libcxx/test/libcxx/containers/unord/next_pow2.pass.cpp
3rdparty/libcxx/libcxx/test/std/utilities/template.bitset/bitset.members/count.pass.cp

### CODE REVIEW

**Comment by anitagov** [comment]
This Passed with sandbox-run.
https://oe-jenkins.eastus.cloudapp.azure.com/blue/organizations/jenkins/OpenEnclave-sandbox/detail/OpenEnclave-sandbox/286/pipeline

File: tests/libcxx/CMakeLists.txt [hunk 1]
```diff
@@ -20,18 +20,20 @@ endif()
...
             continue()
         endif()
     endif()
 
-# few functions invoked in these tests are not considered const expression in gnu
-    if ("${name}" MATCHES "convert_file_time.sh" OR "${name}" MATCHES "sequences_array_at.pass")
-	if (CMAKE_CXX_COMPILER_ID MATCHES GNU)
-	    continue()
+# few functions invoked in these tests are not supported in gnu -- Skip running these tests
+    if ("${name}" MATCHES "convert_file_time.sh" OR "${name}" MATCHES "sequences_array_at.pass"
```
Comment (andschwa) [inline tests/libcxx/CMakeLists.txt]: Just a style nit pick: break before this `OR` too.

File: tests/libcxx/enc/CMakeLists.txt [hunk 2]
```diff
@@ -79,10 +79,12 @@ foreach(testcase ${alltests})
 	endif()
     endif()
 
-# few functions invoked in these tests are not considered const expression in gnu
-    if ("${name}" MATCHES "convert_file_time.sh" OR "${name}" MATCHES "sequences_array_at.pass")
-	if (CMAKE_CXX_COMPILER_ID MATCHES GNU)
-	    continue()
+# few functions invoked in these tests are not supported in gnu and excluded from GCC builds
+    if ("${name}" MATCHES "convert_file_time.sh" OR "${name}" MATCHES "sequences_array_at.pass"
```
Comment (andschwa) [inline tests/libcxx/enc/CMakeLists.txt]: Same nit.

File: tests/libcxx/CMakeLists.txt (hunk 1 above)
Comment (anitagov) [reply tests/libcxx/CMakeLists.txt]: fixed.

File: tests/libcxx/enc/CMakeLists.txt (hunk 2 above)
Comment (anitagov) [reply tests/libcxx/enc/CMakeLists.txt]: fixed

**Pull request merged**